DDOT_module/child_classes/global_noise_matrix_creator.py

```diff
@@ -0,0 +1,119 @@
+"""
+Created on 29.04.2021
+
+@author: Filip Maciejewski
+@contact: filip.b.maciejewski@gmail.com
+"""
+import numpy as np
+from typing import Optional
+
+
+class GlobalNoiseMatrixCreator:
```

Comment:
_Global_ in the name means that it affects whole system, correct?

Reply:
Yes, exactly, nothing to do with programming